src/test/java/nablarch/test/core/http/RestTestSupportTest.java

```diff
@@ -73,6 +75,52 @@ public void testDelegateMethod(@Mocked final DbAccessTestSupport dbSupport) {
             assertTableEquals("message", "sheet", "id");
             assertTableEquals("message", "sheet", "id", true);
         }
+
+        /**
+         * レスポンス内容の文字列比較。
+         */
+        @Test
+        public void testWritingToBodyBuffer() {
+            
+            HttpResponse res = new HttpResponse();
+            assertThat("0", is(res.getContentLength()));
+            
+            res.write("Hello world!\n");
+
+            // レスポンスボディ確認
+            assertThat("Hello world!\n", is(getBodyString(res)));
+
+            // toString確認
+            String toStr = res.toString();
+            assertThat(toStr, containsString("HTTP/1.1 200 OK"));
+            assertThat(toStr, containsString("Content-Length: 13"));
+            assertThat(toStr, containsString("Content-Type: text/plain;charset=UTF-8"));
+            assertThat(toStr, containsString("Hello world!"));
+        }
+
+        /**
+         * ストリームにbyte配列を書き出し、toStringで内容を確認する。
+         */
+        @Test
+        public void testWritingToBodyOutputStream() throws Exception {
+            HttpResponse res = new HttpResponse();
+            String expectedString = "Hello world!\n" + "Hello world2!\n" + "Hello world3!\n";
+            byte[] expectedBytes = expectedString.getBytes();
+
+            res.write(expectedBytes);
+
+            byte[] bytes = new byte[expectedBytes.length];
+            InputStream input = getBodyStream(res);
+            input.read(bytes);
+
+            assertThat(expectedBytes.length, is(bytes.length));
+
+            for(int i = 0; i < expectedBytes.length; i++){
+                if(expectedBytes[i] != bytes[i]) fail();
+            }
+
+            assertThat(res.toString().contains(expectedString), is(true));
```

Comment:
アサートを変更しました。

Reply:
```java
            assertThat(expectedBytes.length, is(actualBytes.length));

            for(int i = 0; i < expectedBytes.length; i++){
                if(expectedBytes[i] != actualBytes[i]) fail();
            }
```

`is()` が配列の内容が完全一致することを検証してくれているので、上記実装も要らなくなるはずです。

http://hamcrest.org/JavaHamcrest/javadoc/1.3/org/hamcrest/CoreMatchers.html#is(T)

> The created matcher provides a special behaviour when examining Arrays, whereby it will match if both the operand and the examined object are arrays of the same length and contain items that are equal to each other (according to the above rules) in the same indexes.
> 作成されたマッチャーは、配列を検査するときに特別な動作を提供します。これにより、オペランドと検査されるオブジェクトの**両方が同じ長さの配列であり**、**同じインデックスに (上記の規則に従って) 互いに等しい項目が含まれている場合に一致します**。

Reply:
不要なテストコードを削除しました。

Reply:
```java
assertThat(expectedBytes.length, is(actualBytes.length));
```

まだ残ってません？
`is()` は配列の長さもチェックするので不要だと思います。

Reply:
削除漏れのテストコードを削除しました。